docs/release-notes/2.16.0.md

```diff
@@ -20,6 +20,15 @@
 
 Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
 
+## Important note for users of PMM 2.16.0
+
+If you started using PMM from version 2.16 and have already upgraded to 2.17, 2.18, or 2.19, you might have some problems with PMM Server monitoring, Remove Monitoring, or RDS/Azure monitoring. If you experience a problem, we recommend you upgrade and replace the Docker container by following the official instructions for an upgrade here: <https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/docker.html#upgrade>.
```

Comment:
@rnovikovP Just merged RNs into release branch and wondered if you wanted this extra text here copied to the 2.20 RNs?